content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,54 @@ the maximum value, and the cumulative sum of values, respectively.
 In addition, the Corda metrics endpoint also includes Caffeine cache metrics (`corda_cache_*`),
 Kafka producer and consumer client metrics (`corda_kafka_*`), JVM metrics (`jvm_*`), process metrics (`process_*`),
 and system metrics (`system_*`) provided by the corresponding Micrometer bindings.
+
+#### Database Worker
+
+The database worker is the sole worker with access to the database, apart from the crypto worker, which has its
+own dedicated database. As a result, the activities carried out within the database worker pertain exclusively to the database.
+
+The database worker is responsible for handling and serving persistence requests originating from various Corda worker types,
+such as the flow worker or the MGM worker. The flow persistence requests metrics presented in this section measure:
+
+* The time needed to handle the flow persistence requests.
+* The time spent by the flow persistence requests on Kafka from the moment they were added by flows until they were received
+by the database worker (Kafka lag).
+
+Additionally, there are background processes occurring within the database worker, namely the reconciliations. These
+reconciliations are background processes responsible for ensuring the alignment of Kafka compacted topics with the database,
```

Comment:
There is some duplication here with first sentence i.e. the reconciliations being background processes.